gateway/src/apicast/policy/soap/soap.lua

```diff
@@ -64,7 +76,9 @@ end
 local function soap_action_in_ctype(headers)
   local ctype = headers['Content-Type']
 
-  if ctype and starts_with(ctype, soap_action_ctype) then
+  -- The Content-Type can be a mix of upper and lower-case chars. Convert it to
+  -- include only lower-case chars to be able to compare it.
+  if ctype and starts_with(lower(ctype), soap_action_ctype) then
```

Comment:
I think this would not work for `application/xml+soap ; action=""` because of the whitespace.